README.MD

```diff
@@ -1,9 +1,19 @@
+## GDPR Cookie Consent banner (fork)
```

Comment:
We should clear this up before merge - I suggest a contributors section @mariojankovic or a contributors file, or both.

The content below can probably be put in a changelog and we can bump the major version.

Reply:
Whatever you suggest. I'm not an opensource guy to be honest - I usually fork and adapt stuff to my needs, so if you want to let me know how you want to proceed on this and what I need to do that'd be great. I just didn't want to confuse people with the fork and the CDN being separate. So we'll revert to your readme and just update the options object.

Reply:
I updated the readme to reflect what we discussed. Now there's just a few new lines in comparison to the original doc.